locale/es/LC_MESSAGES/django.po

```diff
@@ -489,3 +489,43 @@ msgstr "¡Registrar Centro!"
 #: templates/home.html:65
 msgid "Register a Donation Center"
 msgstr "Registrá un Centro de Donación"
+
+#: templates/stats/dashboard.html:16
+#, fuzzy
+#| msgid "State"
+msgid "Stats"
+msgstr "Departamento"
```

Comment:
For what I read in the code this string probably is `Estadísticas` the code, minor detail I'll approve it now and you cant later push a hotfix directly to dev